sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/examples/indexmanagement/sync/SampleIndexManagement.java

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.indexmanagement.sync;
+
+import com.azure.cosmos.*;
```

Comment:
code style is to explicitly import. no wildcard import. Could you change your intellji code style